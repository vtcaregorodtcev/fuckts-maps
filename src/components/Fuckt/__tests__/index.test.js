import React from 'react';
import { Fuckt } from '../index';
import renderer from 'react-test-renderer';

it('renders correctly if not selected', () => {
  const tree = renderer
    .create(<Fuckt fuckt={{ text: '1', selected: false }} zoomOut={0}></Fuckt>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly if selected', () => {
  const tree = renderer
    .create(<Fuckt fuckt={{ text: '1', selected: true }} zoomOut={0}></Fuckt>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
