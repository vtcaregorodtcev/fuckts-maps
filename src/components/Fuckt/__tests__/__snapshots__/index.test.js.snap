// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly if not selected 1`] = `
<div
  className="paper"
  style={
    Object {
      "outline": "none",
      "zoom": "100%",
    }
  }
>
  <div
    className="paper__drag-place"
  />
  <div
    className="paper-content"
  >
    <textarea
      autoFocus={true}
      onChange={[Function]}
      value="1"
    />
  </div>
  <i
    aria-label="icon: close"
    className="anticon anticon-close paper__close"
    onClick={[Function]}
    tabIndex={-1}
  >
    <svg
      aria-hidden="true"
      className=""
      data-icon="close"
      fill="currentColor"
      height="1em"
      viewBox="64 64 896 896"
      width="1em"
    >
      <path
        d="M563.8 512l262.5-312.9c4.4-5.2.7-13.1-6.1-13.1h-79.8c-4.7 0-9.2 2.1-12.3 5.7L511.6 449.8 295.1 191.7c-3-3.6-7.5-5.7-12.3-5.7H203c-6.8 0-10.5 7.9-6.1 13.1L459.4 512 196.9 824.9A7.95 7.95 0 0 0 203 838h79.8c4.7 0 9.2-2.1 12.3-5.7l216.5-258.1 216.5 258.1c3 3.6 7.5 5.7 12.3 5.7h79.8c6.8 0 10.5-7.9 6.1-13.1L563.8 512z"
      />
    </svg>
  </i>
</div>
`;

exports[`renders correctly if selected 1`] = `
<div
  className="paper"
  style={
    Object {
      "outline": "dashed 1px cornflowerblue",
      "zoom": "100%",
    }
  }
>
  <div
    className="paper__drag-place"
  />
  <div
    className="paper-content"
  >
    <textarea
      autoFocus={true}
      onChange={[Function]}
      value="1"
    />
  </div>
  <i
    aria-label="icon: close"
    className="anticon anticon-close paper__close"
    onClick={[Function]}
    tabIndex={-1}
  >
    <svg
      aria-hidden="true"
      className=""
      data-icon="close"
      fill="currentColor"
      height="1em"
      viewBox="64 64 896 896"
      width="1em"
    >
      <path
        d="M563.8 512l262.5-312.9c4.4-5.2.7-13.1-6.1-13.1h-79.8c-4.7 0-9.2 2.1-12.3 5.7L511.6 449.8 295.1 191.7c-3-3.6-7.5-5.7-12.3-5.7H203c-6.8 0-10.5 7.9-6.1 13.1L459.4 512 196.9 824.9A7.95 7.95 0 0 0 203 838h79.8c4.7 0 9.2-2.1 12.3-5.7l216.5-258.1 216.5 258.1c3 3.6 7.5 5.7 12.3 5.7h79.8c6.8 0 10.5-7.9 6.1-13.1L563.8 512z"
      />
    </svg>
  </i>
</div>
`;
